pandas/tests/test_generic.py

```diff
@@ -1874,6 +1874,52 @@ def test_pipe_panel(self):
         with tm.assertRaises(ValueError):
             result = wp.pipe((f, 'y'), x=1, y=1)
 
+    def test_align_joins(self):
+        df = DataFrame(np.array([[1., 2.], [3., 4.]]), columns=list('AB'))
+        ts = Series([5., 6., 7.])
+        check = df.align(ts, join='outer', axis=0, broadcast_axis=1)
+        df1 = DataFrame(check[0])
```

Comment:
all of these tests need to go in ``tests/frame/test_axis_select_reinde`` and/or ``tests/series/test_indexing``. right after the other align tests.